modules/ProvMon/Http/Controllers/ProvMonController.php

```diff
@@ -220,16 +220,23 @@ public function modemOnlineStatus($modem, $hostname)
      *
      * @author  Roy Schneider
      * @param Modules\ProvBase\Entities\Modem
+     * @param mixed
      * @return  string
      */
-    private function modemPicture($modem)
+    private function modemPicture($modem, $sysdescr)
     {
+        if (isset($sysdescr) && array_key_exists('measure.System.SysDescr.0', array_dot($sysdescr))) {
```

Comment:
```suggestion
        if (isset($sysdescr['measure']['System']['SysDescr']['0'])) {
```
I think this is easier understandable and also shorter in code. Or are there any objections?